TuneUpTests/packages.config

```diff
@@ -1,10 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="CommonServiceLocator" version="1.0" targetFramework="net472" />
```

Comment:
do you want to try setting up a GitHub build action for this repo to see if it supports package references?

Reply:
Done